testsuite/features/secondary/srv_salt_git_pillar.feature

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@skip_if_github_validation
+@sle_minion
+@scope_salt
+Feature: Salt Master integration with Git Pillar
+
+  Scenario: Preparing Git Pillar configuration for Salt Master
+    When I setup a git_pillar environment on the Salt Master
+    And I wait for "30" seconds
```

Comment:
Could we instead wait until the file exists?

Reply:
Since `systemctl restart salt-master` is executed in the previous step, this "wait" is just for waiting for the Salt master service to be back and running after the restart. The file is created prior to the restart.

Reply:
Then why do we have to lose 30 seconds?

Sorry, I must be missing something, I do not understand.

Reply:
The Salt Master takes some time on starting up, meaning that even if `systemctl restart venv-salt-minion` was called in the previous step, then we need to wait some seconds to make sure that later steps on the following scenarios can work.

If we do not wait some time here, then the next scenarios will fail when trying to get pillar data, as the Salt master is not ready yet.

Reply:
It does not matter what we wait before of checking the existance of that file.
It only matters that this time can shrink or expand depending the performance of the environment where we run it.
While with a sleep(30) you strictly wait 30 seconds.
With the suggestion I gave you, we will wait until the file exist and continue.

Reply:
Just fixed this now so we wait dynamically now! :)